cloudinit/sources/DataSourceAzure.py

```diff
@@ -265,6 +270,11 @@ def get_resource_disk_on_freebsd(port_id) -> Optional[str]:
     return None
 
 
+def datetime_utcnow() -> datetime.datetime:
```

Comment:
Is this helper really needed here? Also wondering if time.time() would be simpler than datetime.datetime.utcnow() and deltas.

Reply:
time() makes more sense and removed the helper!

Thanks!